src/main/resources/liquibase/db.changelog-1.2.xml

```diff
@@ -68,4 +68,36 @@
             <column name="is_validated_email"/>
         </dropColumn>
     </changeSet>
+
+    <changeSet id="1.2.5: Create a table 'favorite_advertisement'" author="lavniuzhenkova">
+        <createTable tableName="favorite_advertisement">
+            <column name="id" type="${type.uuid}" valueComputed="${uuid_function}">
```

Comment:
maybe you should create combine primary key(user_id and advertisement_id) insted of creating independed id

Reply:
that was a good idea, I did it